src/Tools/dotnet-monitor/ConfigurationJsonWriter.cs

```diff
@@ -217,5 +230,22 @@ public void Dispose()
         {
             _writer.Dispose();
         }
+
+        private class JsonSectionHolder : IDisposable
```

Comment:
nit: Since it does more than just holding it and holders typically have the ownership to dispose what they are holding, how about naming it something like `JsonObjectContext` or `JsonObjectSection`.

Reply:
Ahh, I like `[Json]ObjectContext`. I was trying to come up with a name that involved `object` but couldn't come up with something I liked better than `[Json]SectionHolder`